.github/workflows/.test.yml

```diff
@@ -0,0 +1,559 @@
+name: Reusable
```

Comment:
For consistency let's keep workflow name the same as filename and add same comments as windows reusable workflow:

```suggestion
# reusable workflow
name: .test

# TODO: hide reusable workflow from the UI. Tracked in https://github.com/community/community/discussions/12025
```